docs/README.md

```diff
@@ -16,65 +16,62 @@ aliases:
   - /docs/{{ .version }}/README/
 ---
 
-# Stash
+# KubeStash
 
-[Stash](https://stash.run) by AppsCode is a cloud native data backup and recovery solution for Kubernetes workloads. If you are running production workloads in Kubernetes, you might want to take backup of your disks, databases etc. Traditional tools are too complex to set up and maintain in a dynamic compute environment like Kubernetes. Stash is a Kubernetes operator that uses [restic](https://github.com/restic/restic) or Kubernetes CSI Driver VolumeSnapshotter functionality to address these issues. Using Stash, you can backup Kubernetes volumes mounted in workloads, stand-alone volumes and databases. Users may even extend Stash via [addons](https://stash.run/docs/{{< param "info.version" >}}/guides/addons/overview/) for any custom workload.
+[KubeStash](https://kubestash.com) by AppsCode is a cloud native data backup and recovery solution for Kubernetes workloads. If you are running production workloads in Kubernetes, you might want to take backup of your disks, databases etc. Traditional tools are too complex to set up and maintain in a dynamic compute environment like Kubernetes. KubeStash is a Kubernetes operator that uses [restic](https://github.com/restic/restic) or Kubernetes CSI Driver VolumeSnapshotter functionality to address these issues. Using KubeStash, you can backup Kubernetes volumes mounted in workloads, stand-alone volumes and databases. Users may even extend KubeStash via [addons](https://kubestash.com/docs/{{< param "info.version" >}}/guides/addons/overview/) for any custom workload.
 
-Here, we are going to give you an overview of Stash documentation structure.
+Here, we are going to give you an overview of KubeStash documentation structure.
 
```

Comment:
Update the welcome docs. Remove the items that aren't present in the guides docs.